sdk/tests/deqp/functional/gles3/es3fNegativeVertexArrayApiTests.js

```diff
@@ -166,7 +169,7 @@ goog.scope(function() {
 
         }));
 
-        testGroup.addChild(new es3fApiCase.ApiCaseCallback('vertex_attrib_i_pointer', 'Invalid gl.vertexAttribPointer() usage', gl, function() {
+        testGroup.addChild(new es3fApiCase.ApiCaseCallback('vertex_attrib_i_pointer', 'Invalid gl.vertexAttribIPointer() usage', gl, function() {
             bufferedLogToConsole('gl.INVALID_ENUM is generated if type is not an accepted value.');
             gl.vertexAttribIPointer(0, 1, 0, 0, 0);
```

Comment:
@kenrussell , see https://github.com/KhronosGroup/WebGL/pull/1380. Although test cases for vertexAttribIPointer are all passed in Chromium, this is a potential issue for other browser vendors.